lib/hammer_cli_katello/content_view_component.rb

```diff
@@ -0,0 +1,226 @@
+module HammerCLIKatello
+  module ContentViewComponentBase
+    class ComponentCommand < HammerCLIKatello::SingleResourceCommand
+      include OrganizationOptions
+      include CompositeContentViewNameResolvable
+
+      def get_components(composite_content_view_id)
+        component_options = {:composite_content_view_id => composite_content_view_id}
+        components = ::HammerCLIForeman.foreman_resource(:content_view_components).call(:index,
```

Comment:
`--component-content-view` makes the organization options required when it shouldn't if `--composite-content-view-id` is specified. Organization options should only be required if `--composite-content-view` is used.